From af7c86674c052c39361cc216a81992768e007302 Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Wed, 11 May 2022 21:56:20 -0500
Subject: [PATCH 212/261] docs: Make note of i386-pc specific usage of halt
 command

The --no-apm option is only available on the i396-pc target.

Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 docs/grub.texi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/docs/grub.texi b/docs/grub.texi
index 8397febba..773933216 100644
--- a/docs/grub.texi
+++ b/docs/grub.texi
@@ -4567,10 +4567,10 @@ type are omitted, then the partition will be inactive.
 @node halt
 @subsection halt
 
-@deffn Command halt @option{--no-apm}
-The command halts the computer. If the @option{--no-apm} option
-is specified, no APM BIOS call is performed. Otherwise, the computer
-is shut down using APM.
+@deffn Command halt [@option{--no-apm}[
+The command halts the computer. On the i386-pc target, the @option{--no-apm}
+option, or short @option{-n}, is specified, no APM BIOS call is performed.
+Otherwise, the computer is shut down using APM on that target.
 @end deffn
 
 
-- 
2.36.1

